src/main/scala-2.13/bitcoin/package.scala

```diff
@@ -0,0 +1,62 @@
+package com.mathbot.pay
+
+import com.mathbot.pay.btcpayserver.BitcoinNetwork
+import scala.util.Try
+
```

Comment:
from scala 2.11 -> 2.12 Numeric[] added a parseString method, my solution to cross build was to create a seperate package object holding the implicit converter.

There is a bit of duplicated code from copy / paste but will refactor at a later time